autopts/wid/gatt_client.py

```diff
@@ -1354,6 +1396,49 @@ def hdl_wid_150(params: WIDParams):
 
     return True
 
+def hdl_wid_153(params: WIDParams):
+    """
+    Please write to client support feature handle = 'XXXX'O to enable EATT and Robust Caching.
+    Discover all characteristics if needed.
+    """
+
+    pattern = re.compile(r"'([0-9a-fA-F]+)'")
+    params = pattern.findall(params.description)
+    if not params:
+        logging.error("parsing error")
+        return False
+
+    handle = params[0]
+
+    # First read the existing value in Client Supported Features.
+    btp.gatt_cl_read(btp.pts_addr_type_get(), btp.pts_addr_get(), handle)
+
+    stack = get_stack()
+    stack.gatt_cl.wait_for_verify_values()
+    client_support_features = btp.get_verify_values()
+
+    # Set Robust Caching bit in features.
+    robust_caching_bit = 1
+    eatt_bit = 2
+
+    value = int(client_support_features[1]) | robust_caching_bit | eatt_bit
```

Comment:
Ditto; the bits should be defined in e.g. `defs.py` or something, and used here, instead of being defined multiple times